util/log/logflags/logflags.go

```diff
@@ -17,13 +17,53 @@
 
 package logflags
 
-import "flag"
+import (
+	"flag"
+	"fmt"
+	"strconv"
+	"sync"
+)
+
+type atomicBool struct {
+	sync.Mutex
+	b   *bool
+	def bool // default value
+}
+
+func (ab *atomicBool) IsBoolFlag() bool {
+	return true
+}
+
+func (ab *atomicBool) String() string {
+	ab.Lock()
+	defer ab.Unlock()
+	return fmt.Sprintf("%t", *ab.b)
+}
+
+func (ab *atomicBool) Set(s string) error {
+	ab.Lock()
+	defer ab.Unlock()
+	fmt.Printf("%s", s)
```

Comment:
too late to push.

On Fri, Oct 23, 2015 at 12:12 AM, Tamir Duberstein <notifications@github.com
> wrote:

> In util/log/logflags/logflags.go
> <https://github.com/cockroachdb/cockroach/pull/2899#discussion_r42831880>:
>
> > +}
> > +
> > +func (ab *atomicBool) IsBoolFlag() bool {
> > +	return true
> > +}
> > +
> > +func (ab *atomicBool) String() string {
> > +	ab.Lock()
> > +	defer ab.Unlock()
> > +	return fmt.Sprintf("%t", *ab.b)
> > +}
> > +
> > +func (ab *atomicBool) Set(s string) error {
> > +	ab.Lock()
> > +	defer ab.Unlock()
> > +	fmt.Printf("%s", s)
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2899/files#r42831880>.
>